components/brave_sync/brave_sync_prefs.cc

```diff
@@ -163,6 +170,30 @@ void Prefs::SetSyncAccountDeletedNoticePending(bool is_pending) {
   pref_service_->SetBoolean(kSyncAccountDeletedNoticePending, is_pending);
 }
 
+void Prefs::AddLeaveChainDetail(const char* file, int line, const char* func) {
+  std::string details = pref_service_->GetString(kSyncLeaveChainDetails);
+
+#if BUILDFLAG(IS_WIN)
+  base::FilePath::StringType file_path_string(base::ASCIIToWide(file));
+#else
+  base::FilePath::StringType file_path_string(file);
+#endif
```

Comment:
We can just use `base::FilePath::FromASCII` here, its implementation does `base::ASCIIToWide` internally on Windows.

Reply:
thanks, it is cleaner now

https://github.com/brave/brave-core/pull/21767/commits/ced436f8346c7fbdc21982b5f169fcbd9c6668e1